scripts/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM --platform=linux/amd64 python:3.8.12-buster
+
+RUN apt-get update && apt-get install -y git wget
+RUN wget https://go.dev/dl/go1.18.9.linux-amd64.tar.gz -q
+RUN tar -C /usr/local -xzf go1.18.9.linux-amd64.tar.gz
+
+ENV PATH=${PATH}:/usr/local/go/bin
+
+RUN git clone --recurse-submodules https://github.com/COVESA/vehicle_signal_specification
+
+WORKDIR /vehicle_signal_specification
+
+RUN make install && pip install graphql-core==3.2.3 pytest && make all
```

Comment:
They appear to have a complicated Pipenv setup. But they also [generate releases](https://github.com/COVESA/vehicle_signal_specification/releases), could we just pull those?

Reply:
Let me answer my own question: they only added fields for the low voltage battery in August, after the latest release!